### ISSUE TITLE
Weird meshing: Tetras delete boundary layer

### ISSUE THREAD

**Issue by Jawisr** [enGits/engrid#53]
Hi all,
I'm currently trying to mesh a 3D wing and I'm experiencing some trouble. After some surface iterations I got a 0% of change ratio and 8.1% fluctuation ratio (don't actually know what this ratio means). I've not been able to reduce it to lower values. I don't know if this is converged enough.
The problem comes when I try to add tetras after BL creation. Tetras cells are created from the 2D surface instead of the BL (image 1). Sometimes even the tetra meshing removes prismatic cells as shown in image 2. I attach a few images to show it.
Does anyone know where can be the mistake?

![-home-javier-desktop-remake-engrid_remake-v2-remake_2d_4_bl4 egc - engrid - -1_057](https://cloud.githubusercontent.com/assets/12989191/8742500/56dfd6c4-2c66-11e5-9850-1b0f31b27239.png)

![-home-javier-desktop-remake-engrid_remake-v2-remake_2d_4_bl4 egc - engrid - -1_056](https://cloud.githubusercontent.com/assets/12989191/8742498/547dfa5a-2c66-11e5-8053-8667f541a90f.png)

**Comment by ftpronk** [comment]
Hello,
Judging from your console output you are using and old version.  As NETGEN is not supported anymore, I suggest you try pulling and compiling the latest code from the master branch and try again.  The new implementation is more versatile.

Kind regards,
F.

**Comment by Jawisr** [comment]
Hi again ftpronk,
I tried to update a few weeks ago but I had tons of problems since some ubuntu libraries are not updated. I'll have to invest some time to try it again.
Thanks!

**Comment by ftpronk** [comment]
Hello,
If I recall properly, the only package you were missing was a more up-to-date version of CGAL than that shipped with Ubuntu 14.04, so shouldn't be too difficult to solve.
Kind regards,

F.

**Comment by Jawisr** [comment]
Hi F,
I've tried to contact you by message in order to not to open too many issues, but I couldn't. I have a quick question about master branch.
I've noticed that you have increased edges width in surface triangles visualization. This is so annoying when the mesh is fine because everything becomes blue and no details can be see at all. I attach an example to illustrate. 
![-home-javier-desktop-remake-engrid_remake-v5 master_new -remake_2d_2 egc - engrid - -1_059](https://cloud.githubusercontent.com/assets/12989191/8774544/c2e8c8b4-2edf-11e5-8e64-1857df5b107c.png)

Thanks!
